src/sqliteformat.cpp

```diff
@@ -494,338 +590,347 @@ bool SqliteFormat::modifyComponents(const Incidence::Ptr &incidence, const QStri
 }
 
 //@cond PRIVATE
-bool SqliteFormat::Private::modifyCustomproperties(Incidence::Ptr incidence, int rowid,
-                                                   DBOperation dbop,
-                                                   sqlite3_stmt *stmt1, sqlite3_stmt *stmt2)
+bool SqliteFormat::Private::deleteListsForIncidence(int rowid)
+{
+    int rv = 0;
+    int index = 1;
+
+    if (!mDelIncProperties) {
+        const char *query = DELETE_CUSTOMPROPERTIES;
+        int qsize = sizeof(DELETE_CUSTOMPROPERTIES);
+        SL3_prepare_v2(mDatabase, query, qsize, &mDelIncProperties, nullptr);
+    }
+    SL3_reset(mDelIncProperties);
+    index = 1; SL3_bind_int(mDelIncProperties, index, rowid);
```

Comment:
Could myself just have the statements on their own lines.

Reply:
Alright, done like that, no problem.